sql-statements/sql-statement-change-column.md

```diff
@@ -152,7 +152,7 @@ ERROR 8200 (HY000): Unsupported modify column: change from original type decimal
 * Changes of [Reorg-Data](/sql-statements/sql-statement-modify-column.md#reorg-data-change) types on primary key columns are not supported.
 * Changes of column types on partitioned tables are not supported.
 * Changes of column types on generated columns are not supported.
-* Changes of some data types (for example, some TIME, Bit, Set, Enum, and JSON types) are not supported due to the compatibility issues of the `CAST` function's behavior between TiDB and MySQL.
+* Changes from some data types (for example, some TIME, Bit, Set, Enum, and JSON types) to some other types are not supported due to the compatibility issues of the `CAST` function's behavior between TiDB and MySQL.
```

Comment:
```suggestion
* Changes from some data types (for example, TIME, BIT, SET, ENUM, and JSON types) to some other types are not supported due to the compatibility issues of the `CAST` function's behavior between TiDB and MySQL.
```

The text is not very clear if this is only the `TIME` data type or all time related data types.

Maybe link to `data-type-<type>.md` ? 

Maybe link to `functions-and-operators/cast-functions-and-operators.md` ?

Reply:
yes, but we don't have a clear description doc about what's the compatibility issues is. we need someone familiar with that part to write a doc about it.

json/enum column do can modify to be varchar, so we add this one, previously we are saying no modify on those type can be done,